docs/Setup-Dev.md

```diff
@@ -0,0 +1,27 @@
+#Software to install:
```

Comment:
Add space between `#` and title (this is true also for titles below) + remove trailing `:`